src/InvestJob.js

```diff
@@ -0,0 +1,119 @@
+import crypto from 'crypto';
+import localforage from 'localforage';
+
+import { getLogger } from './logger';
+
+const logger = getLogger(__filename.split('/').slice(-1)[0]);
+
+const KEYS_ARRAY = 'sortedHashArray';
+
+/**
+ * Create an object to hold properties associated with an Invest Job.
+ *
+ */
+export default class InvestJob {
+  /* If none exists, init an empty array for the sorted workspace hashes */
+  static async initDB() {
+    const keys = await localforage.getItem(KEYS_ARRAY);
```

Comment:
This is cool! To make sure I'm understanding...  because Electron uses Chromium (?) it's supported? And the benefit of `localforage` over `localStorage` is that you can access the stored data asynchronously?

Reply:
Yep, we have full use of Chromium! And yes, `localforage` supports async and `localStorage` does not. That's pretty important, although our data is very small at this point, so synchronous calls might be okay. 

But the decision to go with `localForage` was more about this:

`localStorage` only supports simple key: value pair data, where the value is a string. We have more structured data we want to store, like actual javascript objects. It could work to wrap everything in `JSON.stringify` and `JSON.parse` in order to meet that localStorage constraint.

But, all modern browsers also support IndexedDB, which is a full-fledged database that can store our objects as-is. IndexedDB might be overkill for us though (at least at this point) and it has a much lower-level API, so `localForage` is a useful wrapper for both storage options. It stores our data with IndexedDB, but has a really simple API just like `localStorage`.

https://localforage.github.io/localForage/

It's such a simple API that it seemed like the right place to start. There will be very little to change if we decide to replace with a different database at any point.

Reply:
Cool, thanks for the explanation. It does sound like just the right thing for our purpose!